piptools/utils.py

```diff
@@ -315,7 +315,7 @@ def get_compile_command(click_ctx: click.Context) -> str:
             else:
                 if isinstance(val, str) and is_url(val):
                     val = redact_auth_from_url(val)
-                if option.name == "pip_args":
+                if option.name == "pip_args_str":
```

Comment:
this doesn't look related to typing

Reply:
Related to https://github.com/jazzband/pip-tools/pull/1322#discussion_r584272705